sdk/storage/storage-queue/README.md

```diff
@@ -106,7 +103,26 @@ const { QueueServiceClient, SharedKeyCredential } = require("@azure/storage-queu
 
 Use the constructor to create an instance of `QueueServiceClient`, passing in the credential, and options to configure the HTTP pipeline (optional).
 
+You can use `DefaultAzureCredential` (or any other `TokenCredential`) from the @azure/identity library:
+
 ```javascript
+const { DefaultAzureCredential } = require("@azure/identity");
```

Comment:
Will `DefaultAzureCredential` just work? Or is there additional setup required? KeyVault requires some az cli incantations to delegate access to a managed service identity, for example... If that's the case, we should include those instructions here as well.

Reply:
I had added the setup in blob readme. 

https://github.com/Azure/azure-sdk-for-js/tree/master/sdk/storage/storage-blob#create-the-blob-service-client

@willmtemple Can you copy the relevant text from the above link?